arch/arm/soc/nordic_nrf5/nrf52/Kconfig.soc

```diff
@@ -26,3 +26,11 @@ config SOC_NRF52840_QIAA
 	select SOC_NRF52840
 
 endchoice
+
+config NRF52_ARM_MPU_ENABLE
```

Comment:
_ENABLE is unnecessary, and makes the define long. The feature is enabled if it is defined to 'y'; 

CONFIG_ARM_MPU_NRF52X=y

This is sufficient to convery, ARM arch MPU in NRF52X is enabled (=y).